pkg/events/queue/queue_mem_list.go

```diff
@@ -0,0 +1,109 @@
+package queue
+
+import (
+	"container/list"
+	"github.com/aquasecurity/tracee/pkg/external"
+	"sync"
+)
+
+type EventQueueMem struct {
+	amountOfEvents     int // number of cached events
+	mutex              sync.Mutex
+	cache              *list.List
+	EventsCacheMemSize int
```

Comment:
Let's explicit the unit of measure here as well

Reply:
fair enough.